test/recipes/90-test_sslapi.t

```diff
@@ -144,6 +119,5 @@ SKIP: {
        "running sslapitest with modified fips config");
 }
 
-ok(run(test(["ssl_handshake_rtt_test"])),"running ssl_handshake_rtt_test");
```

Comment:
Is this intentional removal?

Reply:
The intention was a minimal reproducer so I could debug what was going on.